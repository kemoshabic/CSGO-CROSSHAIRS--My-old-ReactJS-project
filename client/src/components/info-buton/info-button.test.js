import React from 'react';
import { shallow } from 'enzyme';
import { InfoButton } from './info-button.component';

it('should render InfoButton component', () => {
  expect(shallow(<InfoButton />)).toMatchSnapshot();
});