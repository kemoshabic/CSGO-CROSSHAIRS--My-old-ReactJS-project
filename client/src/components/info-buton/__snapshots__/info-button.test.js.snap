// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InfoButton component 1`] = `ShallowWrapper {}`;
